Cargo.toml

```diff
@@ -27,15 +27,11 @@ default-features = false
 version = "0.2.1"
 default-features = false
 
-[dependencies.rustc-serialize]
-optional = true
-version = "0.3.19"
-
 [dependencies.serde]
 optional = true
-version = ">= 0.7.0, < 0.9.0"
+version = "1.0.0"
```

Comment:
Does it still work if you add `default-features = false`?
I think we need that for proper no-std support.

Reply:
It does. serde's `Cargo.toml` states that it's default `std` feature is for implementations of some `std` types. That doesn't touch our trait impls.
Should our std feature also activate serde's? I don't think we need to, because we don't depend on it ourselves. I don't know if someone can depend purely transitively on no_std serde but then also need to serialize std types.

Reply:
Right, I don't think we need to enable serde/std at all here. Features are additive, so if something else enables serde/std, like num-bigint, then serde will have std overall.